components/membership/membership-http-rpc-impl/src/main/kotlin/net/corda/membership/impl/httprpc/v1/MGMRpcOpsImpl.kt

```diff
@@ -265,6 +267,8 @@ class MGMRpcOpsImpl @Activate constructor(
                 invalidInput(holdingIdentityShortHash)
             } catch (e: MembershipPersistenceException) {
                 throw BadRequestException("${e.message}")
+            } catch (e: PatternSyntaxException) {
+                throw BadRequestException("The regular expression's syntax is invalid.")
```

Comment:
```suggestion
            } catch (e: PatternSyntaxException) {
                throw BadRequestException("The regular expression's (${ruleInfo.ruleRegex}) syntax is invalid.")
```
for a slightly better error message.